pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -102,7 +102,10 @@ angular: teacherDashboard
       //  'remove': {method:'DELETE'},
       //  'delete': {method:'DELETE'}
          update: {method:'POST', url: 'v2/sections/:id/update'},
+         // TODO: This isn't very restful
+         allStudents: {method:'GET', url:'v2/sections/:id/students', isArray: true},
          addStudents: {method:'POST', url:'/v2/sections/:id/students', isArray: true},
+         moveStudents: {method:'POST', url:'/sections/:new_section_code/transfers'},
```

Comment:
Thanks! Is there a way to the equivalent of ```CDO.studio_url``` in JS? The usages I've seen for it are in haml code, but when I try ```url: CDO.studio_url("sections/:new_section_code/transfers")```, ```CDO``` is undefined.